QuantumToolbox.jl/time_evolution/Quantum_Fisher_Information_with_automatic_differentiation.qmd

```diff
@@ -0,0 +1,157 @@
+---
+title: Quantum Fisher Information
+author: Orjan Ameye
+date: last-modified
+---
+
+This notebook demonstrates the computation of Quantum Fisher Information (QFI) for a driven-dissipative Kerr Parametric Oscillator using automatic differentiation. The QFI quantifies the ultimate precision limit for parameter estimation in quantum systems.
+
+We import the necessary packages for quantum simulations and automatic differentiation:
+
+```{julia}
+using QuantumToolbox      # Quantum optics simulations
+using DifferentiationInterface  # Unified automatic differentiation interface
+using SciMLSensitivity   # Allows for ODE sensitivity analysis
+using FiniteDiff         # Finite difference methods
+using LinearAlgebra      # Linear algebra operations
+using CairoMakie              # Plotting
+```
+
+## System Parameters and Hamiltonian
+
+The KPO system is governed by the Hamiltonian:
+$$H = -p_1 a^\dagger a + K (a^\dagger)^2 a^2 - G (a^\dagger a^\dagger + a a)$$
+
+where:
```

Comment:
We should leave an empty line after `where:`, otherwise the list is not properly shown.